spec/features/saml/loa3_sso_spec.rb

```diff
@@ -118,15 +118,15 @@
 
       it 'prompts for OTP at sign in' do
         saml_authn_request = auth_request.create(loa3_with_bundle_saml_settings)
-
         visit saml_authn_request
-
         sign_in_live_with_2fa(user)
+        loa3_sp_session(request_url: saml_authn_request)
```

Comment:
Interesting, I added this because I could not get the test to pass at all. The value of `sp_session` was always an empty object.